software/muscle/meta.yml

```diff
@@ -0,0 +1,48 @@
+name: muscle
+description: MUSCLE is a program for creating multiple alignments of amino acid or nucleotide sequences. A range of options are provided that give you the choice of optimizing accuracy, speed, or some compromise between the two
+keywords:
+    - msa
+    - multiple sequence alignment
+tools:
+  - muscle:
+      description: MUSCLE is a multiple sequence alignment tool with high accuracy and throughput
+      homepage: https://www.drive5.com/muscle
+      documentation: http://www.drive5.com/muscle/muscle.html#_Toc81224840
+      doi: "https://pubmed.ncbi.nlm.nih.gov/15034147/"
+      licence: ['http://www.drive5.com/muscle/manual/license.html']
+
+input:
+  - fasta:
+      type: file
+      description: Input sequences for alignment must be in FASTA format
+      pattern: "*.{fasta,fa,fna}"
+
+output:
+  - Aligned FASTA:
```

Comment:
I think the id of each channel should match the `emit`. 
But I'm not sure how strictly we have been checking this in the past. 

@KevinMenden, can/should we lint for that?

Reply:
Have implemented this in meta.yml. Thanks for pointing it out - definitely improves documentation clarity!

Reply:
@grst no we don't, but it would probably be good, have seen something similar lately as well :+1: